activemq-client/src/main/java/org/apache/activemq/util/JMSExceptionSupport.java

```diff
@@ -61,6 +63,9 @@ public static JMSException create(Exception cause) {
         if (cause instanceof JMSException) {
             return (JMSException)cause;
         }
+        if (cause instanceof MaxFrameSizeExceededException) {
+            return new JMSException(cause.getMessage(), "41300");
```

Comment:
I'd include the max frame size exception as the root cause so the root cause exists and is a type of IOException whether server or client

Reply:
done